mifospay/src/main/java/org/mifos/mobilewallet/mifospay/kyc/KYCContract.java

```diff
@@ -8,19 +8,49 @@
 import org.mifos.mobilewallet.mifospay.base.BaseView;
 
 /**
- * Created by ankur on 16/May/2018
+ * This a contract class working as an Interface for UI
+ * and Presenter components of the KYC Architecture.
+ * @author ankur
+ * @since 11/July/2018
  */
 
 public interface KYCContract {
 
     interface KYCDescriptionView extends BaseView<KYCDescriptionPresenter> {
 
+        /**
+         * Defining onFetchLevelSuccess function in the
+         * KYCContract file  to check the  current kyc level achieved by the user
+         * and implementing it in the KYCDescription Presenter
```

Comment:
wrong. onFetchLevelSuccess is implemented in View class of KYC

Reply:
@manddy there are many issues with the documentation. Understand the flow of the Use case properly.